doc: nrf70: Improve dynamic power save (#13396)
nrfconnect/sdk-nrf: doc/nrf/device_guides/working_with_nrf/nrf70/developing/powersave.rst

Dynamic power save feature
==========================

> When the device is in active mode because of either a packet transmission or a packet reception, an inactivity timer is started with a value :ref:`inactivity timer <ug_nrf70_developing_powersave_inactivity_timer>` value, and the device switches to Power Save mode when the timer expires and the Medium Access Control (MAC) layer is idle.

Reviewer: ```suggestion
When the device is in Active mode due to packet transmission or packet reception, an inactivity timer is started with an :ref:`inactivity timer <ug_nrf70_developing_powersave_inactivity_timer>` value, and the device switches to Power Save mode when the timer expires and the Medium Access Control (MAC) layer is idle.
```

This dynamic switching in and out of Power Save modes is called the dynamic power save feature.
This feature is enabled by default.

> This feature helps the device to save power in low traffic scenarios and protects against intermittent traffic bursts.

Reviewer: ```suggestion
It helps the device to save power in low traffic scenarios and protects against intermittent traffic bursts.
```

.. _ug_nrf70_developing_powersave_inactivity_timer:

Inactivity timer
================

The application can configure the inactivity timer based on the traffic profile.

> After transmissions or receptions are completed and if the Medium Access Control (MAC) layer is idle for a period that is equal to the inactivity timer, the MAC switches to the Power Save mode.

Reviewer: ```suggestion
After transmissions or receptions are completed and if the MAC layer is idle for a period that is equal to the inactivity timer, the MAC switches to the Power Save mode.
```
